From b39388d2e77c2821f7bbb4976fb8429dfffb8c6b Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Mon, 12 Oct 2020 19:37:27 +0000
Subject: [PATCH] Don't build VBCSCompiler for net472

Don't build VBCSCompiler for net472 because it can't load
System.Collections.Immutable.  In the non-source-build build, this
package is loaded from the restored package, but with a tarball,
the package comes from previously source-built, which is only built
for net50.  It then tries to fall back to referenceassemblies which
doesn't contain the package.
See https://github.com/dotnet/source-build/issues/1798
---
 src/Compilers/Server/VBCSCompiler/VBCSCompiler.csproj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Compilers/Server/VBCSCompiler/VBCSCompiler.csproj b/src/Compilers/Server/VBCSCompiler/VBCSCompiler.csproj
index 3fde7d7..ec4dd26 100644
--- a/src/Compilers/Server/VBCSCompiler/VBCSCompiler.csproj
+++ b/src/Compilers/Server/VBCSCompiler/VBCSCompiler.csproj
@@ -5,7 +5,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <LargeAddressAware>true</LargeAddressAware>
-    <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <ServerGarbageCollection>true</ServerGarbageCollection>
     <UseAppHost>false</UseAppHost>
-- 
1.8.3.1
